pkg/syncer/syncer.go

```diff
@@ -299,6 +324,55 @@ func (c *Controller) handleErr(err error, i interface{}) {
 	klog.Errorf("Dropping key %q after failed retries: %v", i, err)
 }
 
+const nsDelimiter = "--"
+const nsTemplate = "kcp" + nsDelimiter + "%s" + nsDelimiter + "%s"
+
+type nameTransformer struct {
+	clusterName string
+	namespace   string
+}
+
+func (n *nameTransformer) toCluster() string {
+	return fmt.Sprintf(nsTemplate, n.clusterName, n.namespace)
```

Comment:
(No action required) Why is the cluster name what we want to encode, vs org-workspace-namespace? Does cluster name map to an org+workspace pair?

cc: @ncdc

Reply:
`clusterName` here refers to the logical cluster, and workspaces are a type of logical cluster.  My understanding of orgs is that they'll be another logical cluster that contains `Workspace` resources but that this hierarchy doesn't really matter to a syncer.  A logical cluster/workspace name should be unique enough without the org included.  The only way I think this wouldn't be true is if workspace names are not unique between shards.

Reply:
I saw the slack thread about this, so please disregard my comment :smile: